### ISSUE TITLE
asgs_main.sh:2726:        # FIXME: writeProperties?

### ISSUE THREAD

**Issue by wwlwpd** [StormSurgeLive/asgs#599]

**Comment by wwlwpd** [comment]
@jasonfleming when you get a chance, can you take a look at line `2726` in `asgs_main.sh` and let me know what we should do here? The last part of the `case` statement can probably be cleaned up so it's clear what the intention is. Right now it looks like this,

```
2706          # write the properties associated with asgs configuration to the
2707          # run.properties file
2708          writeScenarioProperties $NOWCASTDIR
2709          RMQMessage "INFO" "$CURRENT_EVENT" "$THIS>$ENSTORM" "$CURRENT_STATE" "$ENSTORM: $THIS: $START $ENSTORM cycle $ADVISORY."
2710          logMessage "$ENSTORM: $THIS: $START $ENSTORM cycle $ADVISORY."
2711          # create links to the OWI files, assuming they already have the
2712          # adcirc 221, 222, etc file name extensions
2713          cd $ENSTORM 2>> ${SYSLOG}
2714          owiFiles=`ls ${HDIR}/${ADVISORY}*.22*`
2715          for file in $owiFiles; do
2716             ext=${file##*.}
2717             if [[ $ext = 22 ]]; then
2718                cp $file fort.${ext} 2>> ${SYSLOG} # copy fort.22
2719             else
2720                ln -s $file fort.${ext} 2>> ${SYSLOG} # symbolically link data
2721             fi
2722          done
2723          CONTROLOPTIONS="$CONTROLOPTIONS --advisorynum $ADVISORY --advisdir $ADVISDIR --scriptdir $SCRIPTDIR --name $ENSTORM --dt $TIMESTEPSIZE --nws $NWS --co     ntroltemplate ${INPUTDIR}/${CONTROLTEMPLATE} --cst $CSDATE --hstime $HSTIME --hsformat $HOTSTARTFORMAT $OUTPUTOPTIONS"
2724          ;;
2725      off)
2726         # don't need to download any data
2727         # FIXME: writeProperties?
2728         ;;
2729      *) # should be unreachable
2730         RMQMessage "EXIT" "$CURRENT_EVENT" "$THIS>$ENSTORM" "FAIL" "BACKGROUNDMET ($BACKGROUNDMET) did not match an allowable value."
2731         fatal "BACKGROUNDMET did not match an allowable value."
2732         ;;

```

**Comment by jasonfleming** [comment]
This was reworked in the PR #655 and these concerns may be resolved. One point of clarification, was this referring to the `off)` case or the `*)` case (or both)?

**Comment by wwlwpd** [comment]
it was referring the the `#FIXME` comment - will review to make sure, but I'm fine closing this based on your comment, thanks!

**Comment by wwlwpd** [comment]
Oh, yes looks like it was referring to the _should be unreachable_ comment. No worries, this has been resolved with #655.
